sim/src/simulator.cpp

```diff
@@ -331,6 +384,11 @@ void Simulator::_execNextStep(ProfileInfo *profInfo)
     pimpl->currTime += pimpl->dt; // advance simulation time
 }
 
+Field2D Simulator::_H() const
```

Comment:
It is (in my very strong opinion) a terrible practice to use names starting with an underscore. Please refrain from using that (it's more or less reserved by compilers for intrinsics etc. when using double underscores).